arangod/StorageEngine/TransactionState.cpp

```diff
@@ -188,8 +187,8 @@ Result TransactionState::addCollection(DataSourceId cid,
         res.reset(TRI_ERROR_TRANSACTION_UNREGISTERED_COLLECTION,
                   std::string(TRI_errno_string(
                       TRI_ERROR_TRANSACTION_UNREGISTERED_COLLECTION)) +
-                      ": " + cname + " [" + AccessMode::typeString(accessType) +
-                      "]");
+                      ": " + std::string{cname} + " [" +
```

Comment:
for all these error messages, it would be better to move to `absl::StrCat(...)` to make them more efficient.

Reply:
Added in: efe727464c8e5114c22a6fc2a3433ece50ac9527